admin-manual/customization/theming.rst

```diff
@@ -54,23 +60,121 @@ don't have to get your hands too dirty. It's in the details where most of the
 complexities are found. Dominion is the result of a cyclic process of testing
 and refining by a large community of users, try not to underestimate that!
 
-AtoM bundles two themes: arDominionPlugin and arArchivesCanadaPlugin. Their
-names follow the naming convention of Symfony 1.x plugins, because that is how
-themes are implemented in AtoM. You may want to read more about Symfony plugins
+AtoM bundles three themes: arDominionPlugin, arDominionB5Plugin and arArchivesCanadaPlugin.
+Their names follow the naming convention of Symfony 1.x plugins, because that is
+how themes are implemented in AtoM. You may want to read more about Symfony plugins
 later following `one of their guides <http://symfony.com/legacy/doc/gentle-introduction/1_4/en/17-Extending-Symfony#chapter_17_plug_ins>`_.
 
-arDominionPlugin is the default theme, i.e. the theme that will be used in a
+arDominionB5Plugin is the default theme, i.e. the theme that will be used in a
 fresh installation. arArchivesCanadaPlugin was developed as an extension of the
 former and the following instructions will show you how to create your custom
 theme as we did with arArchivesCanadaPlugin.
 
-Assuming that you already have AtoM installed in your development environment
-(you can use our :ref:`Vagrant box <dev-env-vagrant>`), let's start
-building the plugin structure from the command line. Our theme is going to be
-called Corcovado (arCorcovadoPlugin). We are going to track its contents with
-git and publish them in a remote repository hosted by GitHub so we can enable
-others to contribute in the development. The repository is open source so you
-can use it for your own reference, see https://github.com/artefactual-labs/atom-theme-corcovado.
+.. TIP::
+
+   The following instructions assume you already have an AtoM development environment set up
+   locally. If not, we have two development-friendly environments for AtoM - see:
+
+   * :ref:`dev-env-vagrant`
+   * :ref:`dev-env-compose`
+
+++++++++++++++++++++++++
+Custom Bootstrap 5 Theme
+++++++++++++++++++++++++
+
+With the new BS5 framework, creating a custom theme that is an extension
+of the default arDominionB5Plugin theme has become significantly easier. A
+skeleton for an empty AtoM theme plugin that extends arDominionB5Plugin without
+any modifications can be found in this `arThemeB5Plugin repository
+<https://github.com/artefactual-labs/arThemeB5Plugin>`_.
+
+Additional steps for tarball installations:
+*******************************************
+
+If not already installed, first `download the node.js binary distributuion
+<https://nodejs.org/en/download>`_ and export the PATH variable.
+
+The tarball is missing two required files for this: copy the
+`package.json <https://github.com/artefactual/atom/blob/stable/2.7.x/package.json>`_
```

Comment:
Sorry a bit late to the party @melaniekung and @fiver-watson. We should add `package-lock. json` to this list of files.

Reply:
hi @jraddaoui, we've discussed internally and decided not to include the `package-lock.json` to this list since the tarball includes build versions, meant to be ready for use and not for development env. however, if you have other reasons for including the `package-lock.json` here, please let us know.

Reply:
We should, these are instructions to re-build pieces of that version and the lock file is important for that process (more than for development). Even if we only allow small version variations in `package.json`, we want to be "sure" the re-build uses the same dependency tree that was tested. There are other reasons too:

https://docs.npmjs.com/cli/v10/configuring-npm/package-lock-json